src/mocking/mock.ts

```diff
@@ -19,7 +19,7 @@ if (process.env.REACT_APP_HEROKU) {
 
 if (mock.minSideArbeidsgiver) {
     fetchMock.get('/min-side-arbeidsgiver/api/organisasjoner', getOrganisasjonerMock(), {
-        delay: 1000,
```

Comment:
Hva er default delay for mocking? Er det ikke mer realistisk med f.eks 200 ms delay?

Reply:
Default er vel 0 antar jeg. Det hadde gitt mening å gi en realistisk delay, men jeg vil også gjerne at ting går superraskt når jeg utvikler lokalt - så kanskje heller ha det som opt-in?
Ang. realistisk delay, kanskje vi kunne kjørt noen enkle tester i prod, og lagt inn responstiden vi får?